Source/WebKit2/WebProcess/Cookies/WebCookieManager.h

```diff
@@ -33,6 +33,7 @@
 #include <stdint.h>
 #include <wtf/Forward.h>
 #include <wtf/Noncopyable.h>
+#include <wtf/Vector.h>
```

Comment:
included to prevent a complain on a wrong number of template arguments 
because forward declaration for Vector is:
template<typename T, size_t inlineCapacity, typename OverflowHandler, size_t minCapacity> class Vector